### PR TITLE
Add documentation for openEuler clients

### LINKED ISSUE #1635: Add documentation for new client support for openEuler
Add documentation for new client support for openEuler

### PR DESCRIPTION
**Pull request by raulillo82**
# WARNING

~Do not merge this PR until https://github.com/uyuni-project/uyuni/pull/6623 is merged!~ - merged!

# Description

Add documentation for openEuler client, WIP. This only applies to uyuni so far.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes #1635

### CODE REVIEW

**Comment by raulillo82** [comment]
Added tarball with regenerated documentation (build/en): https://w3.suse.de/~rosuna/docu_pr_1636.tar.xz

File: modules/client-configuration/pages/clients-openeuler.adoc
Comment (keichwa) [inline modules/client-configuration/pages/clients-openeuler.adoc:16]:
```suggestion
////

```
I commited the previous suggestion by accident.  It is still commented, though.  If possible, please fill in the actual info.  Thanks for your help!

Comment (keichwa) [reply modules/client-configuration/pages/clients-openeuler.adoc:16]: For reference, see https://github.com/uyuni-project/uyuni-docs/pull/1656 (which is about almalinux).

File: modules/client-configuration/pages/clients-openeuler.adoc
Comment (raulillo82) [reply modules/client-configuration/pages/clients-openeuler.adoc]: openEuler is not supported in SUSE Manager (at least not yet), and it is not clear whether it'll be. Is this then still needed? I was told to remove all references in SUSE Manager documentation and only update it for Uyuni.

Comment (keichwa) [reply modules/client-configuration/pages/clients-openeuler.adoc]: Ok, in this case we probably need an ifeval in the `nav-` file to make sure that it will appear in Uyuni documentation only.  Maybe, this is already in and I missed it.

**Comment by raulillo82** [comment]
Please don't merge this till https://github.com/saltstack/salt/pull/62249 is fixed

**Comment by jcayouette** [comment]
Looks like a small conflict. Otherwise looks good. Thank you!

File: modules/client-configuration/pages/clients-openeuler.adoc
Comment (keichwa) [inline modules/client-configuration/pages/clients-openeuler.adoc:4]:
```suggestion
This section contains information about registering clients running {openeuler} operating systems.
```

File: modules/client-configuration/pages/clients-openeuler.adoc
Comment (keichwa) [inline modules/client-configuration/pages/clients-openeuler.adoc:22]:
```suggestion
```

**Review by keichwa — APPROVED** [review]
In master (5.0), we do not want to intro traditional client documentation no more.  Ornela will adjust the tables.  I think this could be done after merging.

**Comment by raulillo82** [comment]
PR changed, edited the description.
FMPOV, this is ready to be merged (but please feel free to review, as the PR was created very long ago, in case something is wrong, or it needs a rebase, etc).

**Comment by raulillo82** [comment]
Committed suggestions from code review. Let me know if anything else is missing (otherwise this blocks Uyuni 2024.01 release expected for next week).

**Comment by keichwa** [comment]
Raúl Osuna ***@***.***> writes:

> Committed suggestions from code review. Let me know if anything else
> is missing (otherwise this blocks Uyuni 2024.01 release expected for
> next week).

From my POV it is now ready to be merged. I probably can do it for you,
if wanted.

-- 
Karl Eichwalder                       SUSE Software Solutions Germany GmbH
                                      Frankenstraße 146, D-90461 Nürnberg
Geschäftsführer: Ivo Totev, Andrew Myers, Andrew McDonald, Boudien Moerman
(HRB 36809, AG Nürnberg)

**Comment by keichwa** [comment]
For merging, I used the recommended command line steps to solve the conflict.  I hope I got it right.  At least, `master` still builds!

**Pull request merged**
